src/wallet/rpcwallet.cpp

```diff
@@ -4030,6 +4038,9 @@ static RPCHelpMan send()
             {"options", RPCArg::Type::OBJ, RPCArg::Optional::OMITTED_NAMED_ARG, "",
                 {
                     {"add_inputs", RPCArg::Type::BOOL, RPCArg::Default{false}, "If inputs are specified, automatically include more if they are not enough."},
+                    {"include_unsafe", RPCArg::Type::BOOL, RPCArg::Default{false}, "Include inputs that are not safe to spend (unconfirmed transactions from outside keys and unconfirmed replacement transactions).\n"
```

Comment:
Did we already consider for spending replacement transactions from our wallets before this PR ? `isRBFOptIn`doesn't seem used in `FundTransaction` path. Unsafe inputs definition doesn't seem to be tied to replaceability only key externality ?